week7/src/Main.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Tema week 7, TreeSet + HashMap
+ *
+ * @Author Florin
+ */
+
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("TreeSet by name");
+        System.out.println("--------------------");
+        Set<Persoane> psName = new TreeSet<Persoane>(new nameComp());
```

Comment:
Glad that the variable's type is the interface. You're following the best practice.